### ISSUE TITLE
Rules damaged

### ISSUE THREAD

**Issue by Boatshoes24** [arkayenro/arkinventory#910]
Had a ticket that was closed and marked as fixed. Opening a new ticket for the same issue after update. Still receiving the same errors, but possibly in a different line of code from last time:

So I'm not getting a lua error, but my rules have broken specifically in my bank. I am able to fix the rules for my bags if the bank is closed and I open the rules and click accept again, however they break again as soon as I open my bank. I'll paste an example of the rules and the chat error I'm getting. This is only an issue since the most recent update. Was working fine before.

Rule examples that are broken on the bank:

ilvl(835,950) and sb() and not outfit() and type("armor","weapon")
outfit("Tank") and not q(6)
These are the chat errors I'm receiving:

ArkInventory: WARNING> ...rface\AddOns\ArkInventoryRules\ArkInventoryRules.lua:899: Usage: GetVoidItemInfo(tab, slot)
ArkInventory: WARNING> Rule 1008 is now flagged as damaged and will no longer be used until repaired
ArkInventory: WARNING> ...rface\AddOns\ArkInventoryRules\ArkInventoryRules.lua:899: Usage: GetVoidItemInfo(tab, slot)
ArkInventory: WARNING> Rule 1004 is now flagged as damaged and will no longer be used until repaired
ArkInventory: WARNING> ...rface\AddOns\ArkInventoryRules\ArkInventoryRules.lua:899: Usage: GetVoidItemInfo(tab, slot)
ArkInventory: WARNING> Rule 1004 is now flagged as damaged and will no longer be used until repaired
ArkInventory: WARNING> ...rface\AddOns\ArkInventoryRules\ArkInventoryRules.lua:899: Usage: GetVoidItemInfo(tab, slot)
ArkInventory: WARNING> Rule 1004 is now flagged as damaged and will no longer be used until repaired
ArkInventory: WARNING> ...rface\AddOns\ArkInventoryRules\ArkInventoryRules.lua:899: Usage: GetVoidItemInfo(tab, slot)
ArkInventory: WARNING> Rule 1003 is now flagged as damaged and will no longer be used until repaired
ArkInventory: WARNING> ...rface\AddOns\ArkInventoryRules\ArkInventoryRules.lua:899: Usage: GetVoidItemInfo(tab, slot)
ArkInventory: WARNING> Rule 1007 is now flagged as damaged and will no longer be used until repaired
ArkInventory: WARNING> ...rface\AddOns\ArkInventoryRules\ArkInventoryRules.lua:899: Usage: GetVoidItemInfo(tab, slot)
ArkInventory: WARNING> Rule 1004 is now flagged as damaged and will no longer be used until repaired
ArkInventory: WARNING> ...rface\AddOns\ArkInventoryRules\ArkInventoryRules.lua:899: Usage: GetVoidItemInfo(tab, slot)
ArkInventory: WARNING> Rule 1006 is now flagged as damaged and will no longer be used until repaired

**Comment by ArgentSun** [comment]
I am having exactly the same issue. Nothing I do repairs my outfit rules.

**Comment by flatluigi** [comment]
Yep, updated and the issue is still there for me -- except now it's breaking on my regular inventory too.

**Comment by Bestadd** [comment]
removed all other add on to confirm but also having the same error 
ArkInventory: WARNING> Rule 1003 is now flagged as damaged and will no longer be used until repaired
ArkInventory: WARNING> ...rface\AddOns\ArkInventoryRules\ArkInventoryRules.lua:899: Usage: GetVoidItemInfo(tab, slot)

This is repeated for all 4 Damaged rules, all of which use the outfit rule function.  I'm not passing any parameters into outfit.  examples follow:

Rule: 1003
outfit()

Rule: 1005
( type("Armor") or type("Weapon") ) and 
 ( 
    ilvl(850,1000)
 ) and not(outfit()or equip( "trinket" )) 

1002
( type("Armor") or type("Weapon") ) and 
 ( 
    ilvl(0,849) ) and not(outfit() or equip( "trinket" ) )

Rule 1011
not(outfit() ) and  equip( "trinket" )

**Comment by arkayenro** [comment]
hopefully it should be fixed in 3.07.22

this error is not happening on my machine, unlike the previous error, so i cant tell if its really fixed or not

**Comment by rickpelletier** [comment]
I was having this issue and upgrading to 3.07.22 does seem to have fixed it.  However, just FYI I also seem to have lost most of my rules.

**Comment by RevengencerAlf** [comment]
For me at least 3.07.22 seems to still be creating this problem.  I am no longer getting errors (I recreated the rules manually early on) but all of my outfit related rules still seem not to be working.  Here's an example of one of them

Rule 1008
Order 5 
Description DPS
outfit( "Raid", "DPS", "Sub", "Assassination", "Combat", "Survival", "BeastMaster", "Ret", "Marksman", "Feral", "Balance","PewPew" )

For what it's worth I've also had an ongoing issue where I have to hit "reload" every time I log in before my rules would properly process no matter what I did.
